docs/control-structures.rst

```diff
@@ -196,6 +196,33 @@ can still return a value to the caller by use of the ``return`` statement.
         }
     }
 
+.. index:: ! literal suffix; suffix call syntax
+.. _function-calls-suffix-calls:
+
+Suffix Calls
+------------
+
+For free functions designated as :ref:`literal suffix<literal_suffixes>` definitions there is
+another way to perform a function call when the argument is a literal.
+The function can be used as a suffix: ``42 suffix``.
```

Comment:
not sure how this is rendered at the end. but I would expect the definition of that suffix function somewhere here.

Reply:
There's no definition. I'll add one, but I generally wanted to avoid just repeating stuff that you can get by following the link. This is just a small stub to direct you there.

Reply:
Example added.